src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+
+export interface AccordionProps {
+    type?: 'base' | 'single' | undefined
```

Comment:
The only option available should be single. Make it 'single' | undefined

Reply:
Done, fixed